apps/monitoring/botkube/botkube.yaml

```diff
@@ -48,11 +48,21 @@ spec:
               allowed:
                 verbs: ["describe", "get", "top"]
                 resources: ["events", "namespaces", "nodes", "pods", "services", "daemonsets", "deployments", "ingresses", "replicasets", "jobs", "configmaps", "helmreleases"]
+          context:
+            rbac:
+              group:
+                type: Static
+                prefix: ""
+                static:
+                  values: ["botkube-plugins-default"]
     rbac:
-      rules:
-        - apiGroups: ["*"]
-          resources: ["events", "namespaces", "nodes", "pods", "services", "daemonsets", "deployments", "ingresses", "replicasets", "jobs", "configmaps", "helmreleases"]
-          verbs: ["get", "watch", "list"]
+      groups:
+        botkube-plugins-default:
+          create: true
+          rules:
+            - apiGroups: ["*"]
+              resources: ["namespaces", "pods", "services", "daemonsets", "deployments", "ingresses", "replicasets", "jobs", "configmaps", "helmreleases"]
+              verbs: ["get", "watch", "list"]
```

Comment:
is it worth adding describe in here, or is it there already?

Reply:
ignore :D my bad

Reply:
Looks like `describe` is include with the verbs we have
<img width="616" alt="image" src="https://github.com/user-attachments/assets/a581d64c-cefd-481d-b201-ae7a786c4ae3" />